### PR TITLE
feat: Add exercise Protein Translation (#77)

### LINKED ISSUE #77: Add protein-translation practice exercise
I would like to add a port to V of the practice exercise `protein-translation`.

### PR DESCRIPTION
**Pull request by keiravillekode**
Function and argument names, and test cases, are taken from
https://github.com/exercism/problem-specifications/blob/main/exercises/protein-translation/canonical-data.json

closes #77

### CODE REVIEW

File: exercises/practice/protein-translation/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "authors": [],
```
Comment (1ethanhansen) [inline exercises/practice/protein-translation/.meta/config.json:2]: issue: missing username

**Review by 1ethanhansen — CHANGES_REQUESTED** [review]
LGTM once you get the credit you deserve :smile:

File: exercises/practice/protein-translation/.meta/config.json (hunk 1 above)
Comment (keiravillekode) [reply exercises/practice/protein-translation/.meta/config.json:2]: Should this be GitHub username or Exercism username?   vlang README.md says `Author's GitHub username`  but I haven't received Exercism reputation points for any pull requests.

(I created my **keiraville** Exercism account first, then found out the corresponding GitHub username was already taken, hence I'm using @keiravillekode on GitHub.)

Comment (1ethanhansen) [reply exercises/practice/protein-translation/.meta/config.json:2]: I believe it should be Github. Have you linked your exercism account with your github? that's necessary to receive the reputation points

Comment (keiravillekode) [reply exercises/practice/protein-translation/.meta/config.json]: https://exercism.org/settings shows

> Github (Handle)
> keiravillekode
> 

https://exercism.org/settings/integrations shows

> GitHub
> Your Exercism and GitHub accounts are linked. 
>

Comment (keiravillekode) [reply exercises/practice/protein-translation/.meta/config.json]: I have updated `config.json` with my GitHub username, it matches my earlier contributions.

Comment (1ethanhansen) [reply exercises/practice/protein-translation/.meta/config.json]: > https://exercism.org/settings shows
> 
> > Github (Handle)
> > keiravillekode
> 
> https://exercism.org/settings/integrations shows
> 
> > GitHub
> > Your Exercism and GitHub accounts are linked.

oh weird... have you asked about that on the forum? sorry I can't help more than that!

**Review by 1ethanhansen — APPROVED** [review]
LGTM!

File: exercises/practice/protein-translation/.meta/config.json (hunk 1 above)
Comment (keiravillekode) [reply exercises/practice/protein-translation/.meta/config.json]: I haven't. Thanks.

https://exercism.org/docs/building/tracks/concept-exercises

> authors: The GitHub username(s) of the exercise's author(s) (required)

> contributors: The GitHub username(s) of the exercise's contributor(s) (optional)

**Pull request merged**